openmetadata-ui/src/main/resources/ui/src/components/common/RichTextEditor/RichTextEditor.tsx

```diff
@@ -47,7 +47,7 @@ const RichTextEditor = forwardRef<EditorContentRef, RichTextEditorProp>(
         const htmlContent = editorRef.current?.editor?.getHTML() ?? '';
         const backendFormat = formatContent(htmlContent, 'server');
 
-        return backendFormat;
+        return backendFormat === '<p></p>' ? '' : backendFormat;
```

Comment:
I feel we can control this inside formatContent itself instead having condition on component end